build.gradle.kts

```diff
@@ -29,7 +29,7 @@ plugins {
     id("io.spring.dependency-management") version "1.1.7"
     kotlin("plugin.spring") version "2.1.20"
 
-    id("com.github.davidmc24.gradle.plugin.avro") version "1.9.1"
+    id("io.github.androa.gradle.plugin.avro") version "0.0.12"
```

Comment:
https://nav-it.slack.com/archives/C60FFACN5/p1743750683940119?thread_ts=1743485919.199069&cid=C60FFACN5

Gammel plugin blir ikke oppdatert lenger (kan forøvrig ikke se den var i bruk heller)

Reply:
Den ble tatt inn når vi skulle lytte på journalhendelser men så gikk vi for å bruke dependency 
`implementation("no.nav.teamdokumenthandtering:teamdokumenthandtering-avro-schemas:$joarkHendelseVersion")`
Vi tenkte det var en fordel om man fikk dependabot-varsel om endringer i skjeamaet

Men jeg glømte bort å fjerne`com.github.davidmc24.gradle.plugin.avro` 😬 

Publiserer pdl en pakke for disse hendelsene? Ulempen med det er ju at man trekker inn en del kode man ev ikke har bruk for

Reply:
Jeg prøvde å se etter det før jeg dro inn avro-filene, men nei de publiserer ikke noe selv.

Men ja, det legger jo til noe kompleksitet her. Er jo kanskje best å dytte dette ut i tilleggsstonader-kontrakter? Kjipeste med å bygge java-klassene i repoet er at man må bygge prosjektet før det kompilerer. Samme om man kjører en `gradle clean`, så må man bygge det igjen 😅

Reply:
Hmm ja, av den grunnen er det kanskje fint å trekke det ut i kontrakter, då får man det nog til å fungere ut av boksen